c-dependencies/js-compute-runtime/builtins/request-response.cpp

```diff
@@ -2384,8 +2384,162 @@ bool Response::redirect(JSContext *cx, unsigned argc, JS::Value *vp) {
   return true;
 }
 
+namespace {
+static bool callbackCalled;
+static bool write_json_to_buf(const char16_t *str, uint32_t strlen, void *out) {
```

Comment:
These don't need to be `static`, as they're already in the anonymous namespace.
```suggestion
bool callbackCalled;
bool write_json_to_buf(const char16_t *str, uint32_t strlen, void *out) {
```